base/server/examples/installation/ca-mldsa.cfg

```diff
@@ -0,0 +1,58 @@
+[DEFAULT]
+pki_instance_name=pki-tomcat
+pki_https_port=8443
+pki_http_port=8080
+pki_server_database_password=Secret.123
+
+[Tomcat]
+pki_ajp_port=8009
+pki_tomcat_server_port=8005
+
+[CA]
+pki_admin_email=caadmin@example.com
+pki_admin_name=caadmin
+pki_admin_nickname=caadmin
+pki_admin_password=Secret.123
+pki_admin_uid=caadmin
+pki_admin_key_type=mldsa
+pki_admin_key_size=44
+pki_admin_key_algorithm=ML-DSA-44
```

Comment:
In this file I'd suggest to use the default key size (i.e. 65) so we can show that people will only need to specify `pki_admin_key_type=mldsa` if they want to use ML-DSA. If we want to test non-default key size I'd suggest to use the `-D` param in `pkispawn` or maybe create a separate sample `.cfg` file.

The `pki_admin_key_algorithm` seems to be redundant since it can be constructed from the key type and key size. What happens if the key alg doesn't match the specified key type & size? Could we auto-generate the key alg?